db/migrate/20150923150410_add_times_purchased_to_book.rb

```diff
@@ -0,0 +1,5 @@
+class AddTimesPurchasedToBook < ActiveRecord::Migration
+  def change
+    add_column :books, :times_purchased, :integer
```

Comment:
This is fine for now.

In the future, you're going to be making `purchase` objects. A `purchase` will belong to a User and a Book. (Both of which will have_many :purchases). Once that happens, we can remove this column and just use `book.purchases.count`.